src/encoded/tests/test_graph.py

```diff
@@ -1,9 +1,13 @@
+import pytest
+
+@pytest.mark.skip(reason='will be addressed in a PYTEST-REFACTOR')
 def test_graph_dot(testapp):
     res = testapp.get('/profiles/graph.dot', status=200)
     assert res.content_type == 'text/vnd.graphviz'
     assert res.text
 
 
+@pytest.mark.skip(reason='will be addressed in a PYTEST-REFACTOR')
 def test_graph_svg(testapp):
     res = testapp.get('/profiles/graph.svg', status=200)
     assert res.content_type == 'image/svg+xml'
```

Comment:
Laurence wrote these, I am not sure about the purpose

Reply:
Oh. Are you okay with these being skipped and fixed in the next fixture-cleanup? I have to figure out where the svg come from and there are no references to them in the code.

Reply:
maybe @forresttanaka knows? I think SVG is his domain?

Reply:
I have no idea what generates graph.svg. It’s strange, even searching the entire code base including snovault shows nothing with search terms I use. I think Laurence most likely knows how this gets generated.

Reply:
these are used to generate schema diagrams.  Code is in snovault/src/snovault/schema_graph.py.

https://www.encodeproject.org/profiles/graph.svg
and
https://www.encodeproject.org/profiles/graph.dot

Reply:
Theses tests are duplicated in snowflakes/tests.... but those don't get run very often which is why I left them in encoded, to prevent regressions.

Reply:
Ohhhh. schema_graph never came up in my searches. Even a single comment in it could have helped.

Reply:
I will put it to the test.

Reply:
@forresttanaka I just searched for graph.svg.  But then, I knew what the endpoint was supposed to do :)

Reply:
I thought that would work, but “graph.svg” only finds test_graph.py — nothing in snovault.